Modules/_ctypes/_ctypes.c

```diff
@@ -5747,6 +5747,22 @@ wstring_at(const wchar_t *ptr, int size)
     return PyUnicode_FromWideChar(ptr, ssize);
 }
 
+static PyObject *
+memoryview_at(void *ptr, Py_ssize_t size, int readonly)
+{
+    if (PySys_Audit("ctypes.memoryview_at", "nni",
+                    (Py_ssize_t)ptr, size, readonly) < 0) {
+        return NULL;
+    }
+    if (size < 0) {
+        PyErr_Format(PyExc_ValueError,
```

Comment:
Actually, do we want to check this before the audit?

Reply:
IMO, people who look at audit reports won't mind having logs of unsuccessful attempts :)